src/games/index.ts

```diff
@@ -9,6 +9,7 @@ import { reversiGameDef } from './reversi';
 import { cornerusGameDef } from './cornerus';
 import { tictactoeplusGameDef } from './tictactoeplus';
 import { fourinarowGameDef } from './fourinarow';
+import { rotaGameDef } from './rota';
 
```

Comment:
Can you run `yarn run format`? Build failed with formatting issues:
https://travis-ci.com/github/freeboardgames/FreeBoardGames.org/builds/160007352